1.24.3

matteobruni opened the pull request:
feat: added external presets to editor

matteobruni committed:
feat: added external presets to editor

ghost commented:
## DeepCode's analysis on [#63d433](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/916b7e1112bc1abeaf423338d46470b82d5adbb8/matteobruni/tsparticles/63d43325d1ad49bfd374e85f4b77d3a5f147973a/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) found:

- :information_source: **1** minor issue. :point_down:
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">'public' is implicit. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/63d43325d1ad49bfd374e85f4b77d3a5f147973a/core/main/src/Core/Container.ts#L33">Container.ts:33</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/63d43325d1ad49bfd374e85f4b77d3a5f147973a/core/main/src/Core/Container.ts#L38">Container.ts:38</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/63d43325d1ad49bfd374e85f4b77d3a5f147973a/core/main/src/Core/Container.ts#L40">Container.ts:40</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/916b7e1112bc1abeaf423338d46470b82d5adbb8/matteobruni/tsparticles/63d43325d1ad49bfd374e85f4b77d3a5f147973a/pr/_/%2Fcore%2Fmain%2Fsrc%2FCore%2FContainer.ts/javascript%2Ftslint%2Fmember-access/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/916b7e1112bc1abeaf423338d46470b82d5adbb8/matteobruni/tsparticles/63d43325d1ad49bfd374e85f4b77d3a5f147973a/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

matteobruni committed:
build: removed empty element, it will be managed by end user

matteobruni committed:
chore(release): published new version

 - ng-particles@2.7.3
 - inferno-particles@1.7.3
 - jquery-particles@1.24.3
 - preact-particles@1.24.3
 - react-tsparticles@1.24.4
 - svelte-particles@1.7.3
 - particles.vue@2.7.3
 - particles.vue3@1.7.3
 - tsparticles-editor@1.24.3
 - tsparticles@1.24.3
 - ng-particles-demo@1.9.3
 - editor-demo@1.9.3
 - inferno-particles-demo@1.7.3
 - jquery-particles-demo@1.9.3
 - tsparticles-demo@1.9.3
 - preact-particles-demo@1.9.3
 - react-tsparticles-demo@1.9.4
 - svelte-demo@1.7.3
 - particles.vue-demo@1.9.3
 - particles.vue3-demo@1.7.3
 - tsparticles-preset-60fps@1.9.3
 - tsparticles-preset-background-mask@1.9.3
 - tsparticles-preset-basic@1.9.3
 - tsparticles-preset-big-circles@1.9.3
 - tsparticles-preset-bouncing@1.9.3
 - tsparticles-preset-bubbles@1.9.3
 - tsparticles-preset-fire@1.9.3
 - tsparticles-preset-font-awesome@1.9.3
 - tsparticles-preset-fountain@1.9.3
 - tsparticles-preset-sea-anemone@1.9.3
 - tsparticles-preset-snow@1.9.3
 - tsparticles-preset-stars@1.9.3
 - tsparticles-shape-bubble@1.9.3
 - tsparticles-shape-confetti@1.9.3
 - tsparticles-shape-heart@1.9.3
 - tsparticles-shape-multiline-text@1.9.3
 - tsparticles-shape-rounded-rect@1.9.3
 - tsparticles-shape-spiral@1.9.3
 - cra-template-particles-typescript@1.7.4
 - cra-template-particles@1.7.4
 - tsparticles-website@1.24.3

matteobruni committed:
build: updated website with new editor presets

matteobruni committed:
build: added website index confetti preset

matteobruni committed:
build: added website index confetti preset, disabled for issues

matteobruni committed:
Merge branch 'main' into v1
